docs/docsite/rst/porting_guides/porting_guide_core_2.19.rst

```diff
@@ -5,24 +5,546 @@
 Ansible-core 2.19 Porting Guide
 *******************************
 
-This section discusses the behavioral changes between ``ansible-core`` 2.18 and ``ansible-core`` 2.19.
+This section discusses the behavioral changes between ansible-core 2.18 and ansible-core 2.19.
 
-It is intended to assist in updating your playbooks, plugins and other parts of your Ansible infrastructure so they will work with this version of Ansible.
+It is intended to assist in updating your playbooks, plugins and other parts of your Ansible infrastructure so they will
+work with this version of Ansible.
 
-We suggest you read this page along with `ansible-core Changelog for 2.19 <https://github.com/ansible/ansible/blob/stable-2.19/changelogs/CHANGELOG-v2.19.rst>`_ to understand what updates you may need to make.
+We suggest you read this page along with
+`ansible-core Changelog for 2.19 <https://github.com/ansible/ansible/blob/stable-2.19/changelogs/CHANGELOG-v2.19.rst>`_
+to understand what updates you may need to make.
 
-This document is part of a collection on porting. The complete list of porting guides can be found at :ref:`porting guides <porting_guides>`.
+This document is part of a collection on porting.
+The complete list of porting guides can be found at :ref:`porting guides <porting_guides>`.
 
 .. contents:: Topics
 
 
+Engine
+======
+
+This release includes an overhaul of the templating system along with a new feature dubbed Data Tagging.
+These changes have wide-ranging positive effects on security, performance, and user experience.
+Backward compatibility has been preserved where practical, but some breaking changes were necessary.
+
+
+Templating
+----------
+
+Template Trust Model Inversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, ansible-core implicitly trusted all string values to be rendered as Jinja templates, but applied an "unsafe"
+wrapper object around strings obtained from untrusted sources (eg, module results).
```

Comment:
```suggestion
wrapper object around strings obtained from untrusted sources (for example, module results).
```

https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#avoid-using-latin-phrases, for better or for worse, prefers to avoid Latin abbreviations.